src/python/pants/backend/go/target_types.py

```diff
@@ -179,6 +180,18 @@ class SkipGoTestsField(BoolField):
     help = "If true, don't run this package's tests."
 
 
+class GoTestExtraEnvVarsField(StringSequenceField):
```

Comment:
Should `PythonTestsExtraEnvVarsField` and this class share a common base class? The help message would be in the base class. This subclass can override the `alias` to add that `test_` prefix.

Reply:
I can do that if that's preferred. Is there an example of something similar?

Reply:
`PrefixedJvmResolveField` and `JvmResolveField`

Reply:
I think this is fine